src/languages/es.ts

```diff
@@ -1890,6 +1890,7 @@ export default {
             alerts: 'Obtén actualizaciones y alertas en tiempo real',
         },
         bookTravel: 'Reservar viajes',
+        bookDemo: 'Book demo',
```

Comment:
Translation requested here - https://expensify.slack.com/archives/C05S5EV2JTX/p1717097809308109

Reply:
`Pedir demostración`

Reply:
thank you! 🙇